File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/request/CompraModificarDetalleRequest.java
package com.besoft.siserp.lgtc.dto.request;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class CompraModificarDetalleRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long id;
	private Long idProducto;
	private Double cantidad;
	private Double cantidadPerfecto;
	private Double cantidadDaniado;
	private Double precioUnitario;
	private Double subTotal;
	private Integer flagActivo;

}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/request/compra-modificar-detalle.request.ts
export class CompraModificarDetalleRequest {
    id: number;
    idProducto: number;
    nomProducto: string;
    cantidad: number;
    cantidadPerfecto: number;
    cantidadDaniado: number;
    precioUnitario: number;
    subTotal: number;
    flagActivo: number;
}